sum_series.py

```diff
@@ -0,0 +1,13 @@
+def series_sum(n):
+    if n == 0:
+        return 0
+    sum = 0
+    for i in range(n):
+        sum += 1 / ((3 * i) + 1)
```

Comment:
Integer division will produce an integer as a result in Python 2. You need to make one of these numbers a `float`